### ISSUE TITLE
Lost Team Member/Merge with another Team

### ISSUE THREAD

**Issue by dirkduplooy** [Chingu-cohorts/voyage-wiki#298]
**_About You_**
- Voyage number (5 or 6): 7
- Team Name: Toucans 02
- Slack Name: dirkduplooy
- Timezone: UCT+2

**_Issue Description & Expected Outcome:_** 
We lost @Cristian Stefan due to personal problems and we also do not have a decision on a PM yet, I do not see my way open to be PM, if possible can you assign another member who would be interested in leading a team or merge us with a current team please, thanks
**_Symptoms:_**

**_Steps to Recreate:_** 

**_Resolution:_**

**Comment by ReardGjoni** [comment]
Hello @dirkduplooy  Toucan member with similar team problems here. I'd love to come to your team and do my best to contribute

**Comment by dirkduplooy** [comment]
It seems @ReardGjoni is waiting on two other teams as well...

**Comment by tropicalchancer** [comment]
@dirkduplooy @ReardGjoni - let me know what you guys decide and I'll make the necessary github changes. :)

**Comment by dirkduplooy** [comment]
@tropicalchancer Ok spoke to @ReardGjoni and it seems he is still awaiting feedback from the two other teams he can possibly be included with

**Comment by dirkduplooy** [comment]
@tropicalchancer 
Ray [8:04 PM]
It's fine you can remove me from the team I don't want to keep you both waiting
It's fine
I'm looking for other teams anyway
So all the best to you both :grinning:

**Comment by tropicalchancer** [comment]
@dirkduplooy ok thanks - if you find anyone else to join the team or someone reaches out, let me know and I'll add that person to the repo 🙏

**Comment by tropicalchancer** [comment]
@dirkduplooy no activity on this ticket so I will close it. 

If you are still looking for a new member, then please make another ticket and in the title explicitly state what you're looking for "tier one, timezone, project type, tech-stack 

🙏
